### DISCUSSION TITLE
worst prints with mesh leveling

### DISCUSSION THREAD

**Discussion by larie438** [mriscoc/Ender3V2S1#417] [Q&A]
No matter what I do, I'm getting worst prints on my Ender 3v2 while using a 5x5 mesh any suggestions?

**Comment by csanii123** [comment]
Did you preheat the bed/nozzle before trying to mesh level ?
After you mesh leveled did you save the mesh ?
Did you enable Activate Levelling menu option to use the mesh on the printer ? (I missed this and was struggling...)
Did you add the M420 S1 code to your gcode so that the printer will use the mesh ?
After I did all this my 1st layers were even better although they were pretty good before mesh leveling.....
